prov/verbs/configure.m4

```diff
@@ -44,6 +44,17 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[verbs_rdmacm_happy=0])
 	      ])
 
+	AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+			   [
+			    #include <infiniband/verbs_exp.h>
+			   ],
+			   [
+			    return (IBV_EXP_DEVICE_ATTR_ODP | IBV_EXP_DEVICE_ODP);
```

Comment:
Perhaps you want

```c
#if defined(IBV_EXP_DEVICE_ATTR_ODP) && IBV_EXP_DEVICE_ATTR_ODP && defined(IBV_EXP_DEVICE_ODP) && IBV_EXP_DEVICE_ODP
```

There's two possible situations:

1. There's only two cases:
   1. The macros are defined (and their values are irrelevant), and the symbols are present
   1. The macros are not defined, and the symbols are not present
1. There's three cases:
    1. The macros are defined to 1 (or some other "true" value), and the symbols are present
    1. The macros are defined to 0 (or some other "false" value), and the symbols are not present
    1. The macros are not defined, and the symbols are not present (e.g., old versions of OFED)

The example I provided above is relevant for the 2nd situation (where there's 3 cases).